### ISSUE TITLE
Closed connection after sqlcl.run();

### ISSUE THREAD

**Issue by adam-sokol** [oracle/oracle-db-tools#41]
I have create script test.sql:
```
SHOW USER
SCRIPT
sqlcl.setStmt(
    "CONNECT test/test@db:1521/BSS;\n" +
    "SHOW USER;\n"
); 
sqlcl.run();
/
PROPMT AFTER SETSTMT;
SHOW USER;
```
I connect to database via sqlcl:
```
sqlcl adam/adam@db:1521/BSS
```
and run this scrip
```
@test.sql
```
Output from script
```
USER is "ADAM"
Connected.
USER is "TEST"
AFTER SETSTMT
USER is "NULL"
```

Why after 'sqlcl.run();' connection to database is closed?

**Comment by bamcgill** [comment]
Reported,Fixed in Dev and tested.  
Will be released in 22.1 timeframe.

**Comment by bamcgill** [comment]
`HR@db193p🍻🍺 >!cat ~/sandbox/test.sql 
SHOW USER
SCRIPT
sqlcl.setStmt(
    "CONNECT hr/oracle@slci375.us.oracle.com:2193/DB193P;\n" +
    "SHOW USER"
); 
sqlcl.run();
/
PROMPT AFTER SETSTMT
SHOW USER

HR@db193p🍻🍺 >@~/sandbox/test.sql 
USER is "HR"
Connected.

USER is "HR"
AFTER SETSTMT
USER is "HR"
HR@slci375.us.oracle.com:2193/DB193P🍻🍺 >`
